docs/markdown/Builtin-options.md

```diff
@@ -231,10 +231,17 @@ available on all platforms or with all compilers:
 | b_pie               | false                | true, false                                                   | Build position-independent executables (since 0.49.0)                          |
 | b_vscrt             | from_buildtype       | none, md, mdd, mt, mtd, from_buildtype, static_from_buildtype | VS runtime library to use (since 0.48.0) (static_from_buildtype since 0.56.0)  |
 
-The value of `b_sanitize` can be one of: `none`, `address`, `thread`,
-`undefined`, `memory`, `leak`, `address,undefined`, but note that some
-compilers might not support all of them. For example Visual Studio
-only supports the address sanitizer.
+† The default and possible values of sanitizers changed in 1.6. Before 1.6 they
+were string values, and restricted to a specific subset of values: `none`,
+`address`, `thread`, `undefined`, `memory`, `leak`, or `address,undefined`. In
+1.6 it was changed to a free form array of sanitizers, which are checked by a
+compiler and linker check. For backwards compatibility reasons
+`get_option('b_sanitize')` continues to return a string value, which is
+guaranteed to match the old string value if the array value contains the same
+values (ie, `['undefined', 'address'] == 'address,undefined`). If a value not
+allowed before 1.6 is used `b_sanitize` will return a string in undefined order.
+In 1.6 `get_option('b_sanitize', format : 2)`, will return a free form array,
```

Comment:
The commit message says that the way to get the enabled sanitizers as an array is `get_option('b_sanitizers')` (which I like more, BTW) but here it is documented to be `get_option('b_sanitize', format : 2)` and this is implemented below. The commit message needs to be updated.

Reply:
Actually, I think the commit message is fine and the implementation should do what the commit message says.

Reply:
Makes sense, also see below discussion. I'll adapt the code so that we don't introduce the kwarg at all for now.